website/blog/2022-11-09-CCM-Active-solution/ccm-active-cloud-cost-solution.md

```diff
@@ -0,0 +1,85 @@
+
+![auto-ss.png](./assets//autostopping_ss.png)
+
+### Perspectives and Budgets
+
+[Perspectives](https://docs.harness.io/category/e7k0qds7tw-ccm-perspectives)** are the best way to view the correlation of cost data across clouds, clusters, and labels. Perspectives organise multiple resources allocated to a team through multiple cloud providers in one place. Suppose a QA team has access to GCP, and Azure for running tests and quality checks. A QA Perspective would bundle up these resources in a single place where they could be analysed, maintained and regulated together. So, instead of managing cloud costs based on providers, perspectives enable tracking at a team level.
+
+![ss_perspectives.png](./assets/ss_perspectives.png)
+
+[Budgets](https://docs.harness.io/category/lpq8glhiyc-ccm-budgets)** are used to assist perspectives through control checks/limits attached to a team’s allocated cloud resource. Alerts could be sent when the cost exceeds (or is forecasted to exceed) the set budget.
+
+All budgets created in Harness First Generation CMM will be available automatically in Next Generation CCM.
+
+![ss_budgets.png](./assets/ss_budgets.png)
+
+### Enhanced Business Intelligence Dashboards
+
+[Harness Dashboards](https://harness.io/blog/harness-dashboards) helps in measuring software delivery performance. These BI Dashboards are powered by Looker. You may use a built-in dashboard or create a custom dashboard as per requirement. These dashboards are available for data across all the modules. Harness gives you the ability to create your own dashboard to access the key metrics that drives software delivery outcomes.
+
+These Dashboards comes with robust reporting ability, you can set alerts based on preset metrics and schedule reports.
+
+At Harness, we have been working on dashboards that allow you to identify bottlenecks, inform operations, and help drive business decisions. The information and actual metrics may vary based on how engineering teams develop their product.
+
+![dashboard ss.jpeg](./assets/bi-dashboard-ss.png)
+
+# Conclusion
+
+This blog explained how [Harness CCM](https://docs.harness.io/article/rr85306lq8-continuous-efficiency-overview) is an active cloud cost solution, and how it’s at sentry duty with rising cloud cost. To know more about your cloud spending and set up your own cloud cost petrol unit, visit [harness.io](https://harness.io/products/cloud-cost).
+
+In case you are stuck somewhere, need some assistance or want to talk about cloud cost in general, join #cloud-cost-management channel in [harness slack community.](https://join.slack.com/t/harnesscommunity/shared_invite/zt-1jdk336xf-uTvxK5Dg6fElGKTsPv3SUw)
```

Comment:
```suggestion
In case you are stuck somewhere, need some assistance, or want to talk about cloud cost in general, join #cloud-cost-management channel in [Harness Community Slack](https://join.slack.com/t/harnesscommunity/shared_invite/zt-1jdk336xf-uTvxK5Dg6fElGKTsPv3SUw).
```